apps/spruce/src/pages/task/taskTabs/TaskHistory/TestFailureSearchInput/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useMemo, useState } from "react";
+import styled from "@emotion/styled";
+import { palette } from "@leafygreen-ui/palette";
+import TextInput from "@leafygreen-ui/text-input";
+import debounce from "lodash.debounce";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { useQueryParam } from "hooks/useQueryParam";
+import { TaskHistoryOptions } from "../types";
+
+const { red } = palette;
+interface Props {
+  numMatchingResults: number;
+}
+export const TestFailureSearchInput: React.FC<Props> = ({
+  numMatchingResults,
+}) => {
+  const [failingTest, setFailingTest] = useQueryParam<string | undefined>(
+    TaskHistoryOptions.FailingTest,
+    "",
```

Comment:
nit. can just be `string`
```suggestion
  const [failingTest, setFailingTest] = useQueryParam<string>(
    TaskHistoryOptions.FailingTest,
    "",
```